### DISCUSSION TITLE
adjustable dividend rate needs reference to some sort of formula or benchmark by which it's adjusted

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1409] [Securities (SEC)]
But this is *adjustable* rate so needs reference to some sort of formula by which it's adjusted. the explanatoryNote even says "is set by a predetermined formula "

_Originally posted by @rivettp in https://github.com/edmcouncil/fibo/pull/1130#discussion_r480402994_

**Comment by ElisaKendall** [comment]
We agreed in the FCT to add a reference to the benchmark on the class AdjustableRateDividend, rather than on the class PreferredShareWithAdjustableRateDividend.  The class for the share is linked to the dividend, and the dividend is linked to the benchmark.  

We will discuss this in the FCT after the OMG technical meeting.  If you would plan to be on our call on 9/21, that would be helpful.
